kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -316,17 +329,39 @@ def _init_and_run_proof(claim: KClaim) -> tuple[bool, list[str] | None]:
 
             return passed, failure_log
 
-    results: list[tuple[bool, list[str] | None]]
-    if workers > 1:
-        with ProcessPool(ncpus=workers) as process_pool:
-            results = process_pool.map(_init_and_run_proof, claims)
-    else:
-        results = []
-        for claim in claims:
-            results.append(_init_and_run_proof(claim))
+    _LOGGER.info(f'Extracting claims from file: {spec_file}')
+
+    all_claims = kevm.get_claims(
+        spec_file,
+        spec_module_name=spec_module,
+        include_dirs=include_dirs,
+        md_selector=md_selector,
+        claim_labels=None,
+        exclude_claim_labels=exclude_claim_labels,
+    )
+
+    if all_claims is None:
+        raise ValueError(f'No claims found in file: {spec_file}')
+
+    all_claims_by_label: dict[str, KClaim] = {c.label: c for c in all_claims}
+    graph = {c.label: [f'{spec_module_name}.{d}' for d in c.dependencies] for c in all_claims}
+    topological_sorter = graphlib.TopologicalSorter(graph)
```

Comment:
This seems to me that it won't work for spces that are split into multiple modules? Because it assumes that the `depends` clause name not qualified with the module name?